src/DetailsView/details-view-initializer.ts

```diff
@@ -292,6 +294,14 @@ if (isNaN(tabId) === false) {
                 AxeInfo.Default.version,
             );
 
+            const toolData = createToolData(
```

Comment:
The reason I used another variable here for toolData instead of using environmentInfoProvider.getToolData() is to make it easier to remove the environmentInfoProvider as mentioned in the PR description.

Note: you might find similar issue in other files in this PR